pkg/zfs/v1alpha1/zfs_util.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package zfs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	apis "github.com/openebs/zfs-localpv/pkg/apis/openebs.io/core/v1alpha1"
+	"k8s.io/kubernetes/pkg/util/mount"
+)
+
+// ZFSUtil is an empty struct type
+type ZFSUtil struct{}
```

Comment:
This is same as the last comment. Check if it is good to have below
```go
type ZFSVolume struct {
  *apis.ZFSVolume
}
```

Reply:
instead can we just do `type ZFSVolume apis.ZFSVolume`

Reply:
removed it.